tests/integration/test_select.py

```diff
@@ -246,10 +250,15 @@ def test_singular_column_projection_simple(c):
         ["a", "b", "d"],
     ],
 )
-def test_multiple_column_projection(c, input_cols):
+def test_multiple_column_projection(c, parquet_ddf, input_cols):
```

Comment:
Don't think we need `parquet_ddf` as a fixture here? We use the copy in the context fixture but never actually refer to the Python object itself.

Reply:
I've updated the test to also assert df equality and use `parquet_ddf`